src/realm/impl/sequential_getter.hpp

```diff
@@ -67,9 +67,9 @@ class SequentialGetter : public SequentialGetterBase {
         // Set m_leaf_ptr to point at the leaf that contains the value at column row `index`. Return whether or not
         // the leaf has changed (could be useful to know for caller).
 
-        // FIXME: Below line has been commented away because array leafs might relocate during the lifetime of the 
+        // FIXME: Below line has been commented away because array leafs might relocate during the lifetime of the
         // object that owns this SequentialGetter. Enable again when we have proper support for that.
-//        if (index >= m_leaf_end || index < m_leaf_start) 
+        //        if (index >= m_leaf_end || index < m_leaf_start)
```

Comment:
This one doesn't seem right. The difference is "comments" vs "commented out code".